packages/flame_tiled/test/tiled_test.dart

```diff
@@ -404,25 +404,32 @@ void main() {
         Vector2(16, 16),
       );
 
+      final world = World(children: [component]);
+      final cameraComponent = CameraComponent(world: world);
+      cameraComponent.viewfinder.anchor = Anchor.topLeft;
+
       // Need to initialize a game and call `onLoad` and `onGameResize` to
       // get the camera and canvas sizes all initialized
-      final game = FlameGame(children: [component]);
-      component.onLoad();
-      component.onGameResize(mapSizePx);
-      game.onGameResize(mapSizePx);
-      game.camera.snapTo(Vector2(150, 20));
+      final game = FlameGame(children: [world, cameraComponent]);
+      await game.ready();
+      cameraComponent.viewfinder.position = Vector2(150, 20);
     });
 
     test('component size', () {
       expect(component.tileMap.destTileSize, Vector2(16, 16));
       expect(component.size, mapSizePx);
     });
 
-    test('renders', () async {
-      final pngData = await renderMapToPng(component);
+    // TODO(Erick): Don't skip when it is solved.
```

Comment:
Issue :D

Reply:
https://github.com/flame-engine/flame/issues/2548
Will you assign yourself to that one, since you said you would look into it? :)